db/seeds.rb

```diff
@@ -10,6 +10,8 @@
 
 raise StandardError, "Not for production use" if Rails.env.production?
 
+ActiveRecord::Base.connection.execute("DELETE FROM ht_web.otis_approval_requests")
```

Comment:
This I'm guessing is not directly related to the logo stuff, rather testing, but that's fine.